server/commands/battle.py

```diff
@@ -0,0 +1,597 @@
+            client.battle = ClientManager.Battle_char(client, arg.lower())
+            send_info_fighter(client)
+        else:
+            client.send_ooc("No fighter has this name!")
+
+
+def ooc_cmd_info_fighter(client, arg):
+    """
+    Send info about your fighter.
+    Usage: /info_fighter
+    """
+    if client.battle is not None:
+        send_info_fighter(client)
+    else:
+        client.send_ooc("You have to choose a fighter first!")
+
+
+def ooc_cmd_create_fighter(client, arg):
+    """
+    Allow you to create a fighter and to customize its stats.
+    Usage: /create_fighter FighterName HP ATK DEF SPA SPD SPE
+    """
+    args = arg.split(" ")
+    if len(args) < 8:
+        if len(args) > 6:
+            if (
+                args[1].isdigit()
+                and args[2].isdigit()
+                and args[3].isdigit()
+                and args[4].isdigit()
+                and args[5].isdigit()
+                and args[6].isdigit()
+            ):
+                if (
+                    float(args[1]) > 0
+                    and float(args[2]) > 0
+                    and float(args[3]) > 0
+                    and float(args[4]) > 0
+                    and float(args[5]) > 0
+                    and float(args[6]) > 0
+                ):
+                    with open(
+                        "storage/battlesystem/battlesystem.yaml", "r", encoding="utf-8"
+                    ) as c_load:
+                        char = yaml.safe_load(c_load)
+                        args[0] = args[0].lower()
+                        if args[0] not in char:
+                            char[args[0]] = {}
+                            char[args[0]]["HP"] = float(args[1])
+                            char[args[0]]["ATK"] = float(args[2])
+                            char[args[0]]["DEF"] = float(args[3])
+                            char[args[0]]["SPA"] = float(args[4])
+                            char[args[0]]["SPD"] = float(args[5])
+                            char[args[0]]["SPE"] = float(args[6])
+                            char[args[0]]["Moves"] = []
+                            with open(
+                                "storage/battlesystem/battlesystem.yaml",
+                                "w",
+                                encoding="utf-8",
+                            ) as c_save:
+                                yaml.dump(char, c_save)
+                                client.send_ooc(f"{args[0]} has been created!")
+                        else:
+                            client.send_ooc("This fighter has already been created.")
+                else:
+                    client.send_ooc(
+                        "hp, atk, def, spa, spd, spe have to be greater than zero!"
+                    )
+            else:
+                client.send_ooc("Some argument is not an integer.")
+        else:
+            client.send_ooc("Not enough arguments...")
+    else:
+        client.send_ooc("Too many arguments...")
```

Comment:
if if if if else else else else staircase is very hard to read, please revise

Reply:
Done!